app/src/main/res/layout/bottom_sheet_favicons_prompt.xml

```diff
@@ -0,0 +1,114 @@
+            android:orientation="vertical"
+            app:layout_constraintGuide_percent="0.9" />
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            app:layout_constraintTop_toTopOf="parent"
+            android:layout_marginTop="40dp"
+            app:layout_constraintWidth_max="@dimen/autofillBottomSheetContentMaxWidth"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintStart_toStartOf="@id/guidelineStart"
+            app:layout_constraintEnd_toEndOf="@id/guidelineEnd">
+
+            <ImageView
+                android:id="@+id/faviconPromptHeader"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:gravity="center"
+                android:orientation="horizontal"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                android:importantForAccessibility="no"
+                app:srcCompat="@drawable/ic_icons_download_128" />
+
+            <com.duckduckgo.common.ui.view.text.DaxTextView
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:id="@+id/faviconsPromptTitle"
+                android:breakStrategy="balanced"
+                android:text="@string/faviconsPromptTitle"
+                android:layout_marginTop="@dimen/keyline_5"
+                android:gravity="center_horizontal"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/faviconPromptHeader"
+                app:typography="h1" />
+
+
+            <com.duckduckgo.common.ui.view.text.DaxTextView
+                android:id="@+id/faviconsPromptMessage"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/keyline_5"
+                android:text="@string/faviconsPromptMessage"
+                android:textAlignment="center"
+                app:typography="body2"
+                app:layout_constraintEnd_toEndOf="@id/faviconsPromptPrimaryCta"
+                app:layout_constraintStart_toStartOf="@id/faviconsPromptPrimaryCta"
+                app:layout_constraintTop_toBottomOf="@id/faviconsPromptTitle" />
+
+            <com.duckduckgo.common.ui.view.button.DaxButtonPrimary
+                android:id="@+id/faviconsPromptPrimaryCta"
+                android:layout_width="0dp"
+                app:buttonSize="large"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/keyline_5"
+                android:text="@string/faviconsPromptPrimaryCta"
```

Comment:
ditto, we shouldn't be accessing strings or any other resource from another module